packages/framework/fluid-framework/README.md

```diff
@@ -56,7 +88,6 @@ Use of Microsoft trademarks or logos in modified versions of this project must n
 
 <!-- Links -->
 
-[ifluidcontainer]: https://fluidframework.com/docs/apis/fluid-static/ifluidcontainer/
-[sharedmap]: https://fluidframework.com/docs/apis/map/sharedmap/
-[shareddirectory]: https://fluidframework.com/docs/apis/map/shareddirectory/
-[sharedstring]: https://fluidframework.com/docs/apis/sequence/sharedstring/
+[ifluidcontainer]: https://fluidframework.com/docs/api/v2/fluid-framework/ifluidcontainer-interface
```

Comment:
oh? v2?

Reply:
Redirects aren't working yet, so we need to point to v2 specifically for now.

Reply:
Right right.